packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-checkbox/example.html

```diff
@@ -77,3 +77,9 @@ <h2>Example</h2>
     Toggle Pizza field
   </button>
 </gux-button-slot>
+
+<style>
+  gux-form-field-checkbox {
+    margin-bottom: var(--gse-core-spacing-4xs);
+  }
```

Comment:
This looks like a breaking change to me. Could this be added to the `:host` styling?

Reply:
@daragh-king-genesys This is just to add some spacing on the example page.

Reply:
I know but that means anyone who is using the component will have to add some spacing so that is a breaking change.

Reply:
I think this PR should be closed and there should be a new `form-field-checkbox-beta` component created with the checkbox tickets I have been assigned here. https://inindca.atlassian.net/browse/COMUI-2785